### DISCUSSION TITLE
Port VSCode variants to Window Terminal

### DISCUSSION THREAD

**Discussion by DaBultz** [daltonmenezes/aura-theme#159] [Port request]
I'd like to port the variants currently available in VSCode to Windows Terminal, as of right now only the default was ported to Windows Terminal. 

specifically I'm talking porting:
- Aura Dark (Soft Text)
- Aura Soft Dark 
- Aura Soft Dark (Soft Text)

**Comment by daltonmenezes** [comment]
@DaBultz You can work on this for sure, thanks 💜
